l10n_it_fatturapa_in/tests/data/ITNREGCM80H30D612D_20003.xml

```diff
@@ -0,0 +1,132 @@
+                <Provincia>FI</Provincia>
+                <Nazione>IT</Nazione>
+            </Sede>
+        </CedentePrestatore>
+
+	<CessionarioCommittente>
+		<DatiAnagrafici>
+			<CodiceFiscale>04006460275</CodiceFiscale>
+			<Anagrafica>
+				<Denominazione>BIZZ SRL</Denominazione>
+			</Anagrafica>
+		</DatiAnagrafici>
+		<Sede>
+			<Indirizzo>VIA TORINO 135</Indirizzo>
+			<CAP>30100</CAP>
+			<Comune>Venezia</Comune>
+			<Provincia>VE</Provincia>
+			<Nazione>IT</Nazione>
+		</Sede>
+	</CessionarioCommittente>
+
+  </FatturaElettronicaHeader>
+
+
+<FatturaElettronicaBody>
+
+	       <DatiGenerali>
+
+            <DatiGeneraliDocumento>
+                <TipoDocumento>TD01</TipoDocumento>
+                <Divisa>EUR</Divisa>
+                <Data>2020-10-25</Data>
+                <Numero>003</Numero>
+
+                <DatiRitenuta>
+                    <TipoRitenuta>RT01</TipoRitenuta>
+                    <ImportoRitenuta>1.15</ImportoRitenuta>
+                    <AliquotaRitenuta>11.50</AliquotaRitenuta>
+                    <CausalePagamento>R</CausalePagamento>
+                </DatiRitenuta>
+
+                <DatiRitenuta>
+                    <TipoRitenuta>RT04</TipoRitenuta>
+                    <ImportoRitenuta>0.85</ImportoRitenuta>
+                    <AliquotaRitenuta>8.50</AliquotaRitenuta>
+                    <CausalePagamento>R</CausalePagamento>
+                </DatiRitenuta>
+
+        <DatiCassaPrevidenziale>
+          <TipoCassa>TC07</TipoCassa>
+          <AlCassa>8.50</AlCassa>
+          <ImportoContributoCassa>0.85</ImportoContributoCassa>
+          <ImponibileCassa>10.00</ImponibileCassa>
+          <AliquotaIVA>22.00</AliquotaIVA>
+          <Ritenuta>SI</Ritenuta>
+        </DatiCassaPrevidenziale>
+
+       <ImportoTotaleDocumento>10.20</ImportoTotaleDocumento>
+
+            </DatiGeneraliDocumento>
+        </DatiGenerali>
+
+        <DatiBeniServizi>
+            <DettaglioLinee>
+                <NumeroLinea>1</NumeroLinea>
+                <Descrizione>PROVVIGIONI</Descrizione>
+                <PrezzoUnitario>10.00</PrezzoUnitario>
+                <PrezzoTotale>10.00</PrezzoTotale>
+                <AliquotaIVA>22.00</AliquotaIVA>
+                <Ritenuta>SI</Ritenuta>
+            </DettaglioLinee>
+            <DatiRiepilogo>
+                <AliquotaIVA>22.00</AliquotaIVA>
+                <ImponibileImporto>10.00</ImponibileImporto>
+                <Imposta>2.20</Imposta>
+                <EsigibilitaIVA>I</EsigibilitaIVA>
+            </DatiRiepilogo>
+        </DatiBeniServizi>
+
+
+    <DatiPagamento>
+      <CondizioniPagamento>TP02</CondizioniPagamento>
+      <DettaglioPagamento>
+        <ModalitaPagamento>MP05</ModalitaPagamento>
+        <DataScadenzaPagamento>2020-09-30</DataScadenzaPagamento>
+        <ImportoPagamento>10.20</ImportoPagamento>
+        <IstitutoFinanziario>FINECO BANK</IstitutoFinanziario>
+        <IBAN>IT37F0301503200000005796533</IBAN>
```

Comment:
Ciao @eLBati ,
sulla v8 l'IBAN indicato nei test XML risulta errato secondo il metodo `default_iban_check` del modulo `base_iban`, con il seguente messaggio di errore:
_"The IBAN does not seem to be correct. You should have entered something like this ITkk KBBB BBSS SSSC CCCC CCCC CCC Where B = National bank code, S = Branch code, C = Account No, K = Check digit"_
Ho provato anche ad usare un IBAN checker online e risulta effettivamente errato.
Su v12 non viene rilevato?

Reply:
Sembra di no.
Però si può ovviamente correggere (anche nel file ITNREGCM80H30D612D_20004.xml)